src/bundle/Resources/views/themes/admin/content/edit_macros.html.twig

```diff
@@ -0,0 +1,23 @@
+{% macro renderFormField(formField) -%}
```

Comment:
```suggestion
{% macro render_form_field(form_field) -%}
```

Reply:
I don't think this will be backward compatible. Unless we would alias this in `edit_base.html.twig`